### PR TITLE
Updated repo to follow official image guidelines for docker

### PR DESCRIPTION
**Pull request by dpwspoon**

### CODE REVIEW

**Comment by dpwspoon** [comment]
[Link to guidlines](https://docs.docker.com/docker-hub/official_repos/) and [best practices](https://docs.docker.com/articles/dockerfile_best-practices/)

**Comment by dpwspoon** [comment]
Once this is reviewed and merged I'll submit a request to have this become the official kaazing-gateway repo on docker hub.

**Comment by dpwspoon** [comment]
@brennangaunce please review or assign someone to review

File: Dockerfile [hunk 1]
```diff
@@ -1,23 +1,19 @@
 # Pull base image
-FROM ubuntu:14.04
+FROM java:openjdk-8u45-jdk
 
-# Install Zulu Open JDK
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0x219BD9C9
-RUN echo "deb http://repos.azulsystems.com/ubuntu `lsb_release -cs` main" >> /etc/apt/sources.list.d/zulu.list
-RUN apt-get -qq update
-RUN apt-get -qqy install zulu-8=8.6.0.1
+MAINTAINER Kaazing "support@kaazing.com"
```
Comment (brennangaunce) [inline Dockerfile]: This should not be support@kaazing.com.  We should have a mailing list that includes Open Source devs on it.  Support is paid - this is not a paid project - it's a community project, so having support here isn't appropriate.

File: log4j-config.xml [hunk 2]
```diff
@@ -131,8 +131,8 @@
     </logger>
 
     <root>
-        <priority value="warn"/>
-        <appender-ref ref="STDOUT_ERROR"/>        
+        <priority value="info"/>
```
Comment (brennangaunce) [inline log4j-config.xml]: Thank you for checking on this, especially with recent changes to service level logging.

**Comment by dpwspoon** [comment]
@brennangaunce good point on the maintainers, I've updated to be contact via github issues.  Email is not required as far as I can tell

**Comment by dpwspoon** [comment]
They have updated the guides via a blog post on this http://blog.docker.com/2015/06/understanding-official-repos-docker-hub/

**Comment by dpwspoon** [comment]
Adding tests to run tests

**Pull request merged**